pkg/tagger/collectors/ecs_fargate_extract.go

```diff
@@ -89,3 +95,21 @@ func parseECSClusterName(value string) string {
 		return value
 	}
 }
+
+func parseFargateRegion(arn string) string {
+	arnParts := strings.Split(arn, ":")
+	if len(arnParts) < 4 {
+		return ""
+	}
+	if arnParts[0] != "arn" || arnParts[1] != "aws" {
+		return ""
+	}
+	region := arnParts[3]
+
+	// Sanity check
+	if strings.Count(region, "-") != 2 {
```

Comment:
```suggestion
	if strings.Count(region, "-") < 2 {
```

to make it compatible with `us-gov-east-1` 💅

Reply:
Good catch!